pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -329,6 +329,24 @@ static Py_ssize_t get_attr_length(PyObject *obj, char *attr) {
     return ret;
 }
 
+npy_int64 get_long_attr(PyObject *o, const char *attr) {
+  npy_int64 long_val;
+  PyObject *value = PyObject_GetAttrString(o, attr);
+  long_val = (PyLong_Check(value) ?
+              PyLong_AsLongLong(value) : PyInt_AS_LONG(value));
+  Py_DECREF(value);
+  return long_val;
+}
+
+npy_float64 total_seconds(PyObject *td) {
```

Comment:
Getting rid of it would be great.  Neither @jorisvandenbossche nor I know how to call `obj.total_seconds()` in C, and are not inclined to guess.  If there's a quick answer I'll edit.